pkg/share/manager/sql/conversions.go

```diff
@@ -0,0 +1,259 @@
+// Copyright 2018-2021 CERN
```

Comment:
Maybe we can refactor `pkg/cbox/utils/conversions` to reduce duplicity?

Reply:
The differences are big enough that my feeling is it's easier to keep them separate to be honest. Let me know if you want to go for it anyway and I'll see what I can do.